arbitrum-docs/node-running/node-providers.mdx

```diff
@@ -16,23 +15,44 @@ Complete [this form](https://docs.google.com/forms/d/e/1FAIpQLSdw0U-9LcLuih5TZ_Q
 
 This section provides an overview of the available public RPC endpoints for different Arbitrum chains and necessary details to interact with them.
 
-| Name                                   | RPC Url(s)                             | Chain ID | Block explorer                               | Underlying chain | Tech stack       | Sequencer feed URL                    | Sequencer endpoint<sup>⚠️</sup>                  |
+| Name                                   | RPC Url(s)                             | Chain ID | Block explorer                               | Underlying chain | Tech stack       | Sequencer feed URL                    | Sequencer endpoint<sup>⚠️</sup>                   |
 | -------------------------------------- | -------------------------------------- | -------- | -------------------------------------------- | ---------------- | ---------------- | ------------------------------------- | ------------------------------------------------ |
 | Arbitrum One                           | https://arb1.arbitrum.io/rpc           | 42161    | https://arbiscan.io/                         | Ethereum         | Nitro (Rollup)   | wss://arb1.arbitrum.io/feed           | https://arb1-sequencer.arbitrum.io/rpc           |
 | Arbitrum Nova                          | https://nova.arbitrum.io/rpc           | 42170    | https://nova.arbiscan.io/                    | Ethereum         | Nitro (AnyTrust) | wss://nova.arbitrum.io/feed           | https://nova-sequencer.arbitrum.io/rpc           |
 | Arbitrum Goerli (Testnet - Deprecated) | https://goerli-rollup.arbitrum.io/rpc  | 421613   | -                                            | Goerli           | Nitro (Rollup)   | wss://goerli-rollup.arbitrum.io/feed  | https://goerli-rollup-sequencer.arbitrum.io/rpc  |
 | Arbitrum Sepolia (Testnet)             | https://sepolia-rollup.arbitrum.io/rpc | 421614   | https://sepolia.arbiscan.io                  | Sepolia          | Nitro (Rollup)   | wss://sepolia-rollup.arbitrum.io/feed | https://sepolia-rollup-sequencer.arbitrum.io/rpc |
 | Stylus Testnet                         | https://stylus-testnet.arbitrum.io/rpc | 23011913 | https://stylus-testnet-explorer.arbitrum.io/ | Arbitrum Sepolia | Nitro (Rollup)   | wss://stylus-sepolia.arbitrum.io/feed | https://stylus-testnet-sequencer.arbitrum.io/rpc |
 
-⚠️ Unlike the RPC Urls, the Sequencer endpoints only support `eth_sendRawTransaction` and `eth_sendRawTransactionConditional` calls.
 
-⚠️ Important note: Arbitrum public RPCs do not provide Websocket support.
+:::caution 
```

Comment:
Should we compile all these into a list inside one "Caution" admonition? Not sure if that's possible though, but it would make it easier to read this info.

Reply:
Love it, on it